docs/large_cohort_pipeline_readme.md

```diff
@@ -0,0 +1,235 @@
+<br><ins>Configurable inputs:</ins>
+	* NA
+
+8. Frequencies: Generates frequency annotations (AF, AC, AN, InbreedingCoeff) for an input VDS using Hail’s [hl.variant_qc](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.variant_qc) function. The output is written with split multiallelics to `gs://cpg-{dataset}-{access-level}/large-cohort/v0-1/frequencies.ht`.
+<br><ins>Configurable inputs:</ins>
+	* NA
+
+### LoadVqsr Stage
+1. MakeSiteOnlyVcf: This step converts a VDS into a sites-only VCF-ready table.
+<br><ins>Configurable inputs:</ins>
+	* NA
+
+2. Vqsr: This step adds jobs that perform the allele-specific VQSR variant QC
+<br><ins>Configurable inputs:</ins>
+	* [workflow].use_as_vqsr
+	* [workflow].get(‘intervals_path’)
+	* [vqsr].[snp_filter_level]
+	* [vqsr].[indel_filter_level]
+
+3. LoadVqsr: This ste converts VQSR VCFs to HTs.
+<br><ins>Configurable inputs:</ins>
+	* NA
+
+## Notes on what we need
+* all parameters should have options: e.g., Options available are `runner`, `text`, and `json`
+* All thresholds made should have justification as to why that was chosen
+	* Curious as to why `min_pop_prob` was chosen to be 0.5. In gnomad v3.0, this is set to 90% (see https://macarthurlab.org/2019/10/16/gnomad-v3-0/)
+	* Why is 0.2 used as the unrelated threshold? Avuncular pairs and grand-parent/child pairs both have kinship 0.125 in expectation and both have identity-by-descent-zero 0.5 in expectation
+* Any unused values from config files should be removed
+	* scatter_count
+	* max_r_duplication
+	* gencode_gtf = 'gencode/gencode.v39.annotation.gtf.bgz'
+	* references.gnomad section
+* How is get_config()['workflow'] differentiated between the bioheart-test.toml and the large_cohort.toml?
+	* I don’t actually know which toml file arguments are being pulled from, e.g., what differentiates `workflow.sequencing_type` from the `bioheart-test.toml` and `workflow.sequencing_type` from the `genome.toml`?
+* How do you set up the number of workers, cores, etc in a run? Where does `scatter_count` feed into the pipeline?
+* Why does `[workflow].sequencing_type` need to be specified in a separate exome/genome config file? Can this not be added in one of the other toml files?
+* Could the VQSR section in the README and `large_cohort.toml` be described better? I have no idea what these are doing, and why these inputs are chosen. Also, is this only to be specified in the `large_cohort.toml`?
+* Where is `get_config().get()` being pulled from?
+
+## To do:
+- [ ] ensure all correct default values are added in the configurable inputs parameters field
+- [ ] make all descriptions in configurable inputs easy to understand (some of these are still hard to follow)
+- [ ] redistribute settoings in `unsure` configurable inputs section to corresponding toml files
+
+## Configurable inputs
+
+### unsure
```

Comment:
@vladsavelyev @vivbak for all the parameters below, I'm unsure of what config file they're being pulled from. I also don't have all the available info for some of them and not sure of what they're doing. Is it possible to get more info in the `description` section for the ones with `Need info`?